shub/log.py

```diff
@@ -36,13 +36,33 @@
 SHORT_HELP = "Fetch log from Scrapy Cloud"
 
 
+class LogIterFunc(object):
```

Comment:
That was my first intention, but then I found that implied an important refactoring in the core of shub. Also, I don't see the problem with this class. Conserves separation of functions and generality of the core shub functions.